test/fit_tests/deploy/rackhd_package_install.py

```diff
@@ -168,57 +228,30 @@ def test03_install_rackhd_config_files(self):
         os.remove('config.json')
         os.remove('rabbitmq.config')
 
-    def test04_install_rackhd_binary_support_packages(self):
-        print "**** Installing RackHD binaries."
-        self.assertEqual(fit_common.remote_shell(
-            ENVVARS +
-            "mkdir -p /var/renasar/on-tftp/static/tftp;"
-            "cd /var/renasar/on-tftp/static/tftp;"
-            "wget 'https://dl.bintray.com/rackhd/binary/ipxe/monorail-undionly.kpxe';"
-            "wget 'https://dl.bintray.com/rackhd/binary/ipxe/monorail-efi64-snponly.efi';"
-            "wget 'https://dl.bintray.com/rackhd/binary/ipxe/monorail-efi32-snponly.efi';"
-            "wget 'https://dl.bintray.com/rackhd/binary/ipxe/monorail.intel.ipxe';"
-            "wget 'https://dl.bintray.com/rackhd/binary/ipxe/monorail.ipxe';"
-        )['exitcode'], 0, "Binary Support Install failure.")
-        self.assertEqual(fit_common.remote_shell(
-            ENVVARS +
-            "mkdir -p /var/renasar/on-http/static/http/common;"
-            "cd /var/renasar/on-http/static/http/common;"
-            "wget 'https://dl.bintray.com/rackhd/binary/builds/base.trusty.3.16.0-25-generic.squashfs.img';"
-            "wget 'https://dl.bintray.com/rackhd/binary/builds/discovery.overlay.cpio.gz';"
-            "wget 'https://dl.bintray.com/rackhd/binary/builds/initrd.img-3.16.0-25-generic';"
-            "wget 'https://dl.bintray.com/rackhd/binary/builds/vmlinuz-3.16.0-25-generic';"
-        )['exitcode'], 0, "Binary Support Install failure.")
-
-    def test05_reboot_and_check(self):
-        print "**** Reboot and check installation."
-        self.assertEqual(fit_common.remote_shell(
-            "touch /etc/default/on-dhcp-proxy /etc/default/on-http /etc/default/on-tftp /etc/default/on-syslog /etc/default/on-taskgraph"
-        )['exitcode'], 0, "Install failure.")
-        # reboot
-        print "**** Rebooting appliance..."
-        fit_common.remote_shell("shutdown -r now")
-        print "**** Waiting for login..."
-        fit_common.countdown(30)
-        shell_data = 0
-        for dummy in range(0, 30):
-            shell_data = fit_common.remote_shell("pwd")
-            if shell_data['exitcode'] == 0:
-                break
+    def test06_startup(self):
+        print "Restart services."
+        self.assertEqual(fit_common.remote_shell("service isc-dhcp-server restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-http restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-dhcp-proxy restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-syslog restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-taskgraph restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-tftp restart")['exitcode'], 0, "on-http failure.")
```

Comment:
cut-and-paste issue on the failure messages.  All of them say "on-http failure".